schema/aggregations/aggregation_count.js

```diff
@@ -19,7 +22,7 @@ export const AggregationCountType = new GraphQLObjectType({
   },
 });
 
-export default {
+export default ({
   type: AggregationCountType,
   resolve: ({ name, count }, id) => ({ id, name, count }),
-};
+}: GraphQLFieldConfig<AggregationCountType, any>);
```

Comment:
Looking at [this test](https://github.com/facebook/flow/blob/ab146b30bebf93ee25ed95b822f9f7587dacafe5/tests/iterable/map.js#L9-L11) it seems to be a glob for any amount of arguments to `any`.

```js
function mapTest3(map: Map<string, number>): Iterable<*> {
  return map;
}
```

so in our case it could be soaking up unused params, but there's only one ( a 
context.)